### PR TITLE
Merge back

### PR DESCRIPTION
**Pull request by Tjalling-dejong**
## Issue addressed
Fixes #<issue number>

## Explanation
Explain how you addressed the bug/feature request, what choices you made and why.

## General Checklist
- [ ] Updated tests or added new tests
- [ ] Branch is up to date with `main`
- [ ] Tests & pre-commit hooks pass
- [ ] Updated documentation
- [ ] Updated changelog.rst

## Data/Catalog checklist
- [ ] `data/catalogs/predefined_catalogs.yml` has not been modified.
- [ ] None of the old `data_catalog.yml` files have been chagned
- [ ] `data/chagnelog.rst` has been updated
- [ ] new file uses `LF` line endings (done automatically if you used `update_versions.py`)
- [ ] New file has been tested locally
- [ ] Tests have been added using the new file in the test suite

## Additional Notes (optional)
Add any additional notes or information that may be helpful.

### CODE REVIEW

File: docs/api.rst
Comment (savente93) [inline docs/api.rst:1]: The changes in this file should not be included, as we have a new structure in v1

**Review by savente93 — CHANGES_REQUESTED** [review]
We shouldn't include changes from `docs/api.rst` otherwise looks alright

File: docs/api.rst
Comment (Tjalling-dejong) [reply docs/api.rst:1]: I missed that, thanks! I removed it.

**Review by savente93 — APPROVED** [review]
Should be good now!

**Pull request merged**